elastic/kibana
# Telemetry for manual rule run

## Summary

report following events:
- open modal window for manual rule run
- execute manual rule run + save time range in ms
- cancel backfill job
- filter in event log by run type
- show source event date range

Epic - https://github.com/elastic/security-team/issues/2840

### How to test
enable feature flag - `manualRuleRunEnabled`

You can see feature demo here - https://github.com/elastic/kibana/pull/184500

Check that events appears here after some time - https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/r/s/7YYlg
